fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/FDBDirectory.java

```diff
@@ -412,21 +412,24 @@ private CompletableFuture<Void> loadFileReferenceCacheForMemoization() {
             }
             outMap.put(name, fileReference);
         }, context.getExecutor()).thenAccept(ignore -> {
-            if (LOGGER.isDebugEnabled()) {
-                List<String> displayList = new ArrayList<>(outMap.size());
-                long totalSize = 0L;
-                long actualTotalSize = 0L;
-                for (Map.Entry<String, FDBLuceneFileReference> entry: outMap.entrySet()) {
-                    displayList.add(entry.getKey());
-                    totalSize += entry.getValue().getSize();
-                    actualTotalSize += entry.getValue().getActualSize();
-                }
-                LOGGER.debug(getLogMessage("listAllFiles",
-                        LuceneLogMessageKeys.FILE_COUNT, displayList.size(),
-                        LuceneLogMessageKeys.FILE_LIST, displayList,
-                        LuceneLogMessageKeys.FILE_TOTAL_SIZE, totalSize,
-                        LuceneLogMessageKeys.FILE_ACTUAL_TOTAL_SIZE, actualTotalSize));
+            List<String> displayList = new ArrayList<>(outMap.size());
+            long totalSize = 0L;
+            long actualTotalSize = 0L;
+            for (Map.Entry<String, FDBLuceneFileReference> entry: outMap.entrySet()) {
+                displayList.add(entry.getKey());
+                totalSize += entry.getValue().getSize();
+                actualTotalSize += entry.getValue().getActualSize();
             }
+            final String logMessage = getLogMessage("listAllFiles",
+                    LuceneLogMessageKeys.FILE_COUNT, displayList.size(),
+                    LuceneLogMessageKeys.FILE_LIST, displayList,
+                    LuceneLogMessageKeys.FILE_TOTAL_SIZE, totalSize,
+                    LuceneLogMessageKeys.FILE_ACTUAL_TOTAL_SIZE, actualTotalSize);
+            LOGGER.debug(logMessage);
+
+            System.out.println("****************************************List All Files************************************************");
+            System.out.println(logMessage);
+            System.out.println("******************************************************************************************************");
```

Comment:
Is this some kind of debugging thing you added that should be removed before merge?

Reply:
right this is not supposed to be committed.